src/main/java/teammates/ui/controller/FeedbackSessionStatsPageAction.java

```diff
@@ -29,6 +29,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 false);    
         
         data.sessionDetails = logic.getFeedbackSessionDetails(feedbackSessionName, courseId);
+
+        statusToAdmin = "Load feedback session details:<br>" + "Session name: " + feedbackSessionName;
```

Comment:
This issue is only about InstructorHomePage so I think there is no need to do for other places here. Instead, you can open a new issue to to add `statusToAdmin` to other Actions that don't have it.